main.js

```diff
@@ -0,0 +1,45 @@
+const table = document.querySelector('.table');
+table.innerHTML = `
+    <thead>
+      <th>TODO item</th>
+      <th>User name</th>
+      <th>Completed</th>
+    </thead>
+  `
+const tableBody = document.createElement('tbody');
+tableBody.classList.add('tableBody');
+
+const askToServ = async function(url) {
+  return fetch(url)
+    .then(responce => responce.json());
+}
+
+const getData = async function() {
+  return await Promise.all([askToServ(`https://jsonplaceholder.typicode.com/todos`), askToServ(`https://jsonplaceholder.typicode.com/users`)]);
```

Comment:
створи змінні для `url`
```suggestion
  return await Promise.all([
    askToServ(`https://jsonplaceholder.typicode.com/todos`), 
    askToServ(`https://jsonplaceholder.typicode.com/users`)
  ]);
```